.bazelrc

```diff
@@ -6,4 +6,8 @@ build:windows --compiler="clang-cl" --cxxopt="/std:c++17"
 
 build:macos --cxxopt=-std=c++17
 
-build:linux --cxxopt=-std=c++17
\ No newline at end of file
+build:linux --config=clang_config --cxxopt=-std=c++17
```

Comment:
Let's make this specific and explicit to the Linux install for now:

`build:linux --config=linux_usr_lib_llvm_10_config --cxxopt=-std=c++17`